modelforge/potential/utils.py

```diff
@@ -304,45 +303,32 @@ def _distance_to_radial_basis(
 
 # taken from torchani repository: https://github.com/aiqm/torchani
 def neighbor_pairs_nopbc(
-    mask: torch.Tensor, R: torch.Tensor, cutoff: float
+    padding_mask: torch.Tensor, coordinates: torch.Tensor, cutoff: float
```

Comment:
We should be consistent, positions or coordinates in our function variable names.

Reply:
Thanks! Updated this!